### PR TITLE
move login/register into a form

### PR DESCRIPTION
**Pull request by huyphamily**
It felt weird not having the expected behavior of enter to work on the login/register screen. I wrapped the inputs in a form and added an onSubmit handler. This way, both the actions of clicking the button and pressing enter will submit the post to login/register.

### CODE REVIEW

File: app/containers/LoginOrRegister.jsx [hunk 1]
```diff
@@ -17,28 +17,27 @@ class LoginOrRegister extends Component {
...
-      password
-    }));
+
+    if (isLogin) {
+      dispatch(manualLogin({
+        email,
+        password
+      }));
+    } else {
+      dispatch(signUp({
```
Comment (18601673727) [inline app/containers/LoginOrRegister.jsx]: Not every login request is a register request, so maybe you should do more logic check before dispatching it.

Comment (huyphamily) [inline app/containers/LoginOrRegister.jsx]: I'm keeping the logic the same as the boilerplate. The isLogin property on the state determines whether or not the user is trying to login or register. When it is true, the user is trying to login. When it is false, the user is trying to register. Take a look at the old renderButton method on this container, the logic will look similar. The action creator toggleLoginMode toggles between the two mode, and there is only two mode in this case.

**Comment by choonkending** [comment]
Thanks for contributing!!

I agree that enter not working is an unexpected behavior. Will review and merge hopefully late today!

**Comment by choonkending** [comment]
Thanks again for contributing!

**Comment by huyphamily** [comment]
Not a problem, happy to contribute. You have a very nice boilerplate!

**Pull request merged**